scripts/generate_certs.sh

```diff
@@ -13,10 +13,10 @@ COUNTY=${4:-San Mateo County}
 sudo mkdir /opt/acme_tiny
```

Comment:
I don't use tiny acme anymore (I tend to use certbot) - is this example script worth keeping?

Reply:
I don't think it is worth having. Glancing at the [tiny-acme repository](https://github.com/diafygi/acme-tiny) gives off the impression that it's in maintenance mode.

<img width="844" alt="Screen Shot 2021-04-04 at 2 47 08 AM" src="https://user-images.githubusercontent.com/10052309/113502180-35c38500-94f0-11eb-9408-10a431b53b8b.png">

Certbot on the other hand is still actively being developed. 

In addition, Certbot is backed by the [EFF](https://www.eff.org/pages/eppcode) whereas tiny-acme is owned by an individual.

Reply:
ok! I'm good removing `generate_certs.sh` and `generate_certs-tiny-acme.sh` (the two that don't use certbot). It looks like we have instructions for cerbot in the rendered logs :point_right: https://singularityhub.github.io/sregistry/docs/install/https

Reply:
Removed